int32 goal_marker_id  
string goal_marker_frame
